docs/CHANGELOG.md

```diff
@@ -10,7 +10,8 @@ The format of this change log follows the advice given at [Keep a CHANGELOG](htt
 
 ## [Unreleased]
 ### Changed
-* Updated all uses of `actions/checkout` from `v2` (using node 12) to `v3` (using node 16), because [actions using node 12 are deprecated](https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/) and will sop working in the future.
+- Internal improvements to the release process.
+- Updated all uses of `actions/checkout` from `v2` (using node 12) to `v3` (using node 16), because [actions using node 12 are deprecated](https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/) and will sop working in the future.
```

Comment:
Hi @stronk7,
typo here: `sop` => `stop`.

HTH,
Matteo

Reply:
Fun, for curiosity just looked to the (potential) meaning of sop (as noun and as verb).

Thanks @scara ... amending now... I'm blind!

Reply:
> he used some bread to sop up the sauce

🤣 
In italian, this action looks like `fare [la] scarpetta` i.e. to clean one's plate with a piece of bread 😉 .

HTH,
Matteo

Reply:
😆